underfs/hdfs/pom.xml

```diff
@@ -159,25 +159,20 @@
       </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-dependency-plugin</artifactId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <version>1.8</version>
         <executions>
           <execution>
             <id>copy-installed</id>
             <phase>install</phase>
             <goals>
-              <goal>copy</goal>
+              <goal>run</goal>
             </goals>
             <configuration>
-              <artifactItems>
-                <artifactItem>
-                  <groupId>${project.groupId}</groupId>
-                  <artifactId>${project.artifactId}</artifactId>
-                  <version>${project.version}</version>
-                  <type>${project.packaging}</type>
-                  <overWrite>true</overWrite>
-                </artifactItem>
-              </artifactItems>
-              <outputDirectory>${lib.path}</outputDirectory>
+              <target>
+                <copy todir="${basedir}/../../lib" overwrite="true"
```

Comment:
Cannot do something like `parent.parent.basedir`?

Reply:
unfortunately, `parent.parent.basedir` is not supported by `antrun`